Analytics/Classes/SEGAnalyticsConfiguration.h

```diff
@@ -140,4 +140,30 @@ typedef NSMutableURLRequest *_Nonnull (^SEGRequestFactory)(NSURL *_Nonnull);
  */
 @property (nonatomic, strong, nullable) id<SEGApplicationProtocol> application;
 
+/**
+ * A dictionary of filters to redact payloads before they are sent.
+ * This is an experimental feature that currently only applies to Deep Links.
+ * It is subject to change to allow for more flexible customizations in the future.
+ *
+ * The key of this dictionary should be a regular expression string pattern,
+ * and the value should be a regular expression substitution template.
+ *
+ * By default, this contains a Facebook auth token filter, configured as such:
+ * @code
+ * @"(fb\\d+://authorize#access_token=)([^ ]+)": @"$1((redacted/fb-auth-token))"
+ * @endcode
+ *
+ * This will replace any matching occurences to a redacted version:
+ * @code
+ * "fb123456789://authorize#access_token=secretsecretsecretsecret&some=data"
+ * @endcode
+ *
+ * Becomes:
+ * @code
+ * "fb123456789://authorize#access_token=((redacted/fb-auth-token))"
```

Comment:
given the example `fb123456789://authorize#access_token=secretsecretsecretsecret&some=data`, wouldn't the desired output be `fb123456789://authorize#access_token=((redacted/fb-auth-token))&some=data`? (i.e. `&some=data`  at the end.

Reply:
Facebook sends other private query parameters to the deep link that could be considered sensible, so we remove everything until we get a space (`[^ ]`).

Reply:
Got it - sounds good 👍